cads_catalogue_api_service/contents.py

```diff
@@ -29,12 +29,27 @@
 )
 
 
-def _apply_common_filters(query, site: str, ctype: str | None = None):
-    return query.where(
+def _apply_common_filters(
+    query,
+    site: str,
+    ctype: str | None = None,
```

Comment:
@alfredotoselli as `ctype` now is a QS parameter, I think is better to make it multi-valued (as related datasets)